### PR TITLE
Added schematic and PCB design files for ESP shield.

### PR DESCRIPTION
**Pull request by gunturiCM**
## Main Changes
Added schematic and PCB design files. These files are the design files of a custom ESP32 shield board.
The shield has the MPU9250 IMU, ESCs, and buzzer circuit connections. All of the remaining free pins are brought out as separate ports for later use.

A separate push will be done for the Raspberry Pi shield.

## Associated Issues

- Addresses #57

### CODE REVIEW

**Comment by gunturiCM** [comment]
A separate push will be done for the Raspberry Pi shield.

File: design/PCBs/ESP_BOARD/Gerber_ESP_SHIELD_PCB_2023-06-03/How-to-order-PCB.txt
Comment (alphaville) [inline design/PCBs/ESP_BOARD/Gerber_ESP_SHIELD_PCB_2023-06-03/How-to-order-PCB.txt:1]: Change from `.txt` to `.md`

**Review by alphaville — CHANGES_REQUESTED** [review]
Can we have a figure file (PNG, preferably) with the new board?

File: design/PCBs/ESP_BOARD/Read_me.md
[the remark this answers is not in the document]
Comment (gunturiCM) [reply design/PCBs/ESP_BOARD/Read_me.md:1]: Done.

File: design/PCBs/ESP_BOARD/Gerber_ESP_SHIELD_PCB_2023-06-03/How-to-order-PCB.txt
Comment (gunturiCM) [reply design/PCBs/ESP_BOARD/Gerber_ESP_SHIELD_PCB_2023-06-03/How-to-order-PCB.txt:1]: Done.

**Comment by gunturiCM** [comment]
> Can we have a figure file (PNG, preferably) with the new board?

I have included separate `*.png` files for the schematic, and PCB.

**Comment by jamie-54** [comment]
Should we not keep the PCB the same width as the Pi so they can stick together nicely and fit inside the case? Hoping to make the case smaller in future as well. Also think we would be better moving the  ESC header to a similar place as the old PCB just so we don't have to lengthen wires ect

**Comment by jamie-54** [comment]
@gunturiCM I made the changes I mentioned above can you check you are happy with them and update the PNG when you're happy

**Comment by gunturiCM** [comment]
> @gunturiCM I made the changes I mentioned above can you check you are happy with them and update the PNG when you're happy

I see you have made the board compact. But my intention was to make the IMU be at the centre of the board (Like designing around the IMU, rather than designing around the ESP). I will take your submission as a reference to make the board more compact. Thanks!

**Comment by gunturiCM** [comment]
> @gunturiCM I made the changes I mentioned above can you check you are happy with them and update the PNG when you're happy

@jamie-54 I have re-arranged the components according to your suggestions. Can you have a look and let me know if it needs any changes?

**Comment by gunturiCM** [comment]
@jamie-54 I have changed the `ESC` headers position, please have a look and suggest any required modifications. Thanks.

**Review by jamie-54 — APPROVED** [review]
All looks good to me I've approved @gunturiCM @alphaville are you happy to merge?
Also think we should get one printed as son as possible

**Comment by gunturiCM** [comment]
Added mounting holes. Please have a look.

**Review by alphaville — APPROVED** [review]
Looks good - thanks, Chandra

**Pull request merged**